plugins/kotlin/j2k/shared/src/org/jetbrains/kotlin/nj2k/conversions/StreamConversion.kt

```diff
@@ -0,0 +1,38 @@
+// Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package org.jetbrains.kotlin.nj2k.conversions
+
+import org.jetbrains.kotlin.analysis.api.KtAnalysisSession
+import org.jetbrains.kotlin.nj2k.NewJ2kConverterContext
+import org.jetbrains.kotlin.nj2k.RecursiveConversion
+import org.jetbrains.kotlin.nj2k.callOn
+import org.jetbrains.kotlin.nj2k.tree.*
+
+class StreamConversion(context: NewJ2kConverterContext) : RecursiveConversion(context) {
```

Comment:
For a simple conversion such as this, `BuiltinMembersConversion` should be enough.